### PR TITLE
Us110933/remove top level nav

### PR DESCRIPTION
**Pull request by TomMichielsD2L**
Updated the lesson header so that if the hide-description class is set, it no longer has a click effect or hover effect. Also added a unit test that ensures the element gets the right class when the class is set on the entity.

### CODE REVIEW

File: test/d2l-lesson-header.html [hunk 1]
```diff
@@ -200,6 +200,19 @@
 		});
 
 	});
+
+	describe('with the hide-description class', () => {
```
Comment (jwong-d2l) [inline test/d2l-lesson-header.html]: do you want to add a test for the current-activity behaviour too?

File: components/d2l-lesson-header.js [hunk 2]
```diff
@@ -136,7 +136,10 @@ class D2LLessonHeader extends ASVFocusWithinMixin(CompletionStatusMixin()) {
 			https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12837456/*/
 			background-color: var(--d2l-color-celestine, #006fbf);
 		}
-
+		:host(.hide-description) .d2l-header-lesson-link,
```
Comment (jwong-d2l) [inline components/d2l-lesson-header.js]: So this doesn't remove the link part, it just makes it go nowhere and makes it *look* like not a link?

File: components/d2l-lesson-header.js [hunk 3]
```diff
@@ -308,15 +311,21 @@ class D2LLessonHeader extends ASVFocusWithinMixin(CompletionStatusMixin()) {
 	_getHeaderClass(currentActivity, entity, focusWithin) {
 		const selfLink = entity && entity.getLinkByRel('self').href;
 		const selected = currentActivity === selfLink;
-		return this._getTrueClass(focusWithin, selected);
+		let trueClasses = this._getTrueClass(focusWithin, selected);
```
Comment (jwong-d2l) [inline components/d2l-lesson-header.js]: I really should have called this function something different.

File: components/d2l-lesson-header.js (hunk 2 above)
Comment (TomMichielsD2L) [reply components/d2l-lesson-header.js]: Yes, as per our offline discussion, this does not address keyboard navigation

File: test/d2l-lesson-header.html (hunk 1 above)
Comment (TomMichielsD2L) [reply test/d2l-lesson-header.html]: Added!

File: components/d2l-lesson-header.js (hunk 2 above)
Comment (TomMichielsD2L) [reply components/d2l-lesson-header.js]: Josh has indicated that it is fine that the user can keyboard navigate to the header.

File: test/d2l-lesson-header.html (hunk 1 above)
Comment (TomMichielsD2L) [reply test/d2l-lesson-header.html]: I actually added what you wanted this time

**Pull request merged**